WebRoot/js/zimbra/common/ZLoginFactory.js

```diff
@@ -105,7 +109,12 @@ function(msgs) {
 		showButton : true,
 		buttonName : msgs["login"] || "",
 		
-		copyrightText : msgs["splashScreenCopyright"] || ""
+		copyrightText : msgs["splashScreenCopyright"] || "",
+
+		twoFactorCodeTitle: msgs["twoFactorCodeTitle"] || "",
+		twoFactorCodeLabel: msgs["twoFactorCodeLabel"] + ':' || "",
+		twoFactorCodeButton: msgs["twoFactorCodeButton"] || "",
+		trustDeviceLabel: msgs["trustDeviceLabel"] || "Always trust this device" // @TODO: Remove this when adding translations for this key
```

Comment:
Fix before moving ticket to UAT

Reply:
Currently `trustedDevices` is not implemented, but I have added the HTML for it. I added this comment specifically for when it will be implemented and translations would be added for the same.

Reply:
Removed the code for trustedDevices after discussion with @BhavinBathani